packages/ag-charts-enterprise/src/features/annotations/axisButton.css

```diff
@@ -0,0 +1,31 @@
+.annotations-axis-button {
+    position: absolute;
+    left: 0px;
+    top: 0px;
+    user-select: none;
+    font:
+        15px Verdana,
+        sans-serif;
+    overflow: hidden;
+    white-space: nowrap;
+    z-index: 99998;
```

Comment:
Given the new chart-overlay container, I don't think these massive z-indices are needed any more.

And if we need to order elements within the chart-overlay we should use variables, e.g. `--ag-charts-layer-tooltip: 1;`, so we can coordinate them in the theme -> https://github.com/ag-grid/ag-charts/blob/latest/packages/ag-charts-community/src/styles/styles.css